### PR TITLE
Integrate RCTBPKCalendar with SimpleDate

### PR DESCRIPTION
**Pull request by k0nserv**
Continuing from #83.

### CODE REVIEW

File: ios/Podfile [hunk 1]
```diff
@@ -24,7 +24,7 @@ target 'Backpack Native' do
   pod 'Folly', podspec: '../node_modules/react-native/third-party-podspecs/Folly.podspec'
 
   # RN Bridging Pods
-  pod 'Backpack', '~> 6.3.0'
+  pod 'Backpack', '~> 7.0.0'
```
Comment (shaundon) [inline ios/Podfile]: May as well bump this to 8.0.0 now that's out.

File: packages/react-native-bpk-component-calendar/src/ios/CalendarBridge/RCTBPKCalendarManager.m [hunk 2]
```diff
@@ -42,11 +42,11 @@ - (UIView *)view {
...
 }
 
-RCT_EXPORT_VIEW_PROPERTY(minDate, NSDate)
-RCT_EXPORT_VIEW_PROPERTY(maxDate, NSDate)
+RCT_REMAP_VIEW_PROPERTY(minDate, rct_minDate, NSDate)
+RCT_REMAP_VIEW_PROPERTY(maxDate, rct_maxDate, NSDate)
 RCT_EXPORT_VIEW_PROPERTY(selectionType, BPKCalendarSelection)
 RCT_EXPORT_VIEW_PROPERTY(locale, NSLocale)
-RCT_EXPORT_VIEW_PROPERTY(selectedDates, NSArray<NSDate *> *)
+RCT_REMAP_VIEW_PROPERTY(selectedDates, rct_selectedDates, NSArray<NSDate *> *)
```
Comment (shaundon) [inline packages/react-native-bpk-component-calendar/src/ios/CalendarBridge/RCTBPKCalendarManager.m]: Where do you find documentation for things like this? When I was investigating that calendar issue with RN 0.59, I tried looking up these macros and found nothing. Even the source code wasn't that helpful!

File: ios/Podfile (hunk 1 above)
Comment (k0nserv) [reply ios/Podfile]: Yeah shounds good

File: packages/react-native-bpk-component-calendar/src/ios/CalendarBridge/RCTBPKCalendarManager.m (hunk 2 above)
Comment (k0nserv) [reply packages/react-native-bpk-component-calendar/src/ios/CalendarBridge/RCTBPKCalendarManager.m]: I looked at the source code of React Native itself. For example here's usage in [`RCTViewManger`](https://github.com/facebook/react-native/blob/9bb5c32de9ef06942e38ea6cf5d9000c37741c37/React/Views/RCTViewManager.m#L114)

**Comment by shaundon** [comment]
I fixed the conflicts and bumped the Backpack version to 8.0.0. It's all good now, I think.

**Pull request merged**